services/fastmail/config.json

```diff
@@ -1,10 +1,18 @@
 {
   "config": {
     "name": "fastmail",
-    "label": "Fast, reliable email",
-    "description": "Fast, reliable email for businesses, families and professionals.",
+    "label": "FastMail",
+    "description": "Email, calendars and contacts done right.",
     "category": "email"
   },
+  "fields": [
+    {
+      "name": "dkim",
+      "label": "DKIM Key",
+      "description": "Enter your provided DKIM Key. You can find this under Settings | Domain",
```

Comment:
I've tested this out and the description here is the only part that was a little tricky. What do you think of saying something like this?

> Enter the content of the DKIM record as provided on Fastmail's Domain Settings page for this domain.

I don't know if that's any clearer, but it might help since it's really the entire DNS record we want to set here, instead of just the key.